src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -225,6 +225,11 @@ const PaymentAgentWithdraw = (() => {
                 }
                 break;
         }
+        if (response.error) {
```

Comment:
seems like we were handling error in `default` case above. should we move that code to an error function so we handle them both the same way? or are they different?

Reply:
Yeah, I've changed it. Thanks for mentioning that, I completely missed it.